### ISSUE TITLE
font scaling changed between v12.1 and v12.2

### ISSUE THREAD

**Issue by catmando** [wkhtmltopdf/wkhtmltopdf#2092]
Rendering of svg text for v12.1 matches major browsers, v12.2 does not.

try this on both versions... notice that in v12.2 middle line of text is cut off, and scales differently.  Again this only happens on v12.2 . V12.1 matches rendering on all other browsers.  

Sorry for the long url... 

wkhtmltopdf --debug-javascript  --window-status LOADED -B 0 -R 0 -T 0 -L 0 --page-width 133.350000084 --page-height 184.150000116 "http://doodlebin.catprintlabs.com/v1/backgrounds/121/front?IWUSEV2WK&f%5Bdoogin-text-field-94%5D=Love%2C+Diane%2C+Emma%2C+and+Greg&f%5Bdoogin-text-field-95%5D=Season%E2%80%99s+Greetings&f%5Bdoogin-text-field-96%5D=Wishing+you+and+yours+a+warm+and+merry+%0Aholiday+season.&p%5B0%5D%5Bl%5D=-45.94&p%5B0%5D%5Bs%5D=http%3A%2F%2Fiw2.catprint.com%2Fv1%2Ffiles%2F6cbefa3ebfffda86326b970e20fde414%2Fpanels%2F1.jpg&p%5B0%5D%5Bt%5D=-55.13&p%5B0%5D%5Bw%5D=448.88" foo.pdf

**Comment by PhilterPaper** [comment]
This *may* be the same problem as I'm having (0.12.2.4 patched QT, Win 7 64 bit). I have two HTML documents of about the same size, built on the same framework, using the same CSS. Although `font-size: 36pt` is explicitly given for the title page text (via a class), on one PDF it's about 80% the size as on the other. The smaller (80%) font continues throughout the document. I have been looking for over a day, but can't find any content on the smaller one that's triggering the behavior. In chopping out sections of HTML to see if I could find the offender, sometimes I saw text cut off *in the middle of a letter* at the right margin, although I can't reliably reproduce this. Currently I'm running with `body { width: auto; }` but I've also tried width 100% with no difference. I *do* set font-size to 80% *within* tables, but the entire document is scaled about 80%.

How does body width interact with paper size and wkhtmltopdf-set margins? Is there *anything* in HTML or CSS that is expected to scale the entire output to "fit" something on the page? Of course, I'm checking that Adobe Reader claims page size is 100% for both documents.
